src/cli-constant.js

```diff
@@ -217,43 +229,41 @@ const detailOptions = normalizeDetailOptions({
   },
   "use-tabs": {
     type: "boolean",
-    category: "format",
+    category: CATEGORY_FORMAT,
     forwardToApi: true,
-    description: "Indent lines with tabs instead of spaces."
+    description: "Indent with tabs instead of spaces."
   },
   version: {
     type: "boolean",
-    category: "command",
     alias: "v",
     description: "Print Prettier version."
   },
   "with-node-modules": {
     type: "boolean",
-    category: "config",
+    category: CATEGORY_CONFIG,
     description: "Process files inside 'node_modules' directory."
   },
   write: {
     type: "boolean",
-    category: "command",
```

Comment:
From @azz's comment: 

> `--write` is probably the most used option so it seems odd that it is at the bottom. ([link](https://github.com/prettier/prettier/pull/2767#discussion_r137940445))

> I'd say the following are "directives"/"commands", as they completely change the behaviour of the program:
>
>* `--list-different`
>* `--find-config-path`
>* `--help`
>* `--version`
> ([link](https://github.com/prettier/prettier/pull/2767#discussion_r137940438))

I think it makes sense to have the `command` category, what do you think?

---

Other changes look good to me. 👍

Reply:
I kinda liked that `--write` was at the bottom, because `--help` is longer than what fits on my screen so `--write` actually _is_ the first thing I see :) But I'll look into the categories again.

Reply:
Ah! Haven't thought that the last option will be displayed "first". 😆

Let me know when you want to get this merged.

Reply:
I pushed a commit with a new categorization idea now.